src/main/java/org/commcare/formplayer/api/json/PromptToJson.java

```diff
@@ -35,6 +35,7 @@ public static void parseQuestion(FormEntryPrompt prompt, JSONObject questionJson
         parseCaption(prompt, questionJson);
         questionJson.put("help", jsonNullIfNull(prompt.getHelpText()));
         questionJson.put("binding", jsonNullIfNull(prompt.getQuestion().getBind().getReference().toString()));
+        questionJson.put("question_id", jsonNullIfNull(prompt.getQuestion().getBind().getReference().getNameLast()));
```

Comment:
could pull `prompt.getQuestion().getBind().getReference()` out into a variable to avoid repeating it.

Also, any reason not to return the `question_id` in `toString()` if there is no text?

Reply:
> Also, any reason not to return the question_id in toString() if there is no text?

I am not sure if we would want to always default to question_id whenever there is no text for example in the primary form entry view today we just show empty space whenever the question view is empty. This seems a valid UX to me as it accurately reflects the app buider design. 

<img width="910" alt="Screenshot 2021-08-18 at 2 28 36 PM" src="https://user-images.githubusercontent.com/4679137/129869739-7f065a89-ad3f-4e9a-a047-840bde7ec79f.png">